MAUI/Polar-Charts/Axis/Types.md

```diff
@@ -0,0 +1,340 @@
+
+N> If  minimum or maximum value is set, the other value is calculated by default internally.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfPolarChart.SecondaryAxis>
+    <chart:NumericalAxis Maximum="100" Minimum="0" Interval="20"/>
+</chart:SfPolarChart.SecondaryAxis>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+NumericalAxis secondaryAxis = new NumericalAxis()
+{
+    Maximum = 100,
+    Minimum = 0,
+    Interval = 20
+};
+chart.SecondaryAxis.Add(secondaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Category Axis
+
+The [CategoryAxis]() is an indexed based axis that plots values based on the index of the data point collection. The points are equally spaced here.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfPolarChart>
+. . .
+<chart:SfPolarChart.PrimaryAxis>
+    <chart:CategoryAxis/>
+</chart:SfPolarChart.PrimaryAxis>
+
+</chart:SfPolarChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfPolarChart chart = new SfPolarChart();
+. . .
+CategoryAxis primaryAxis = new CategoryAxis();
+chart.PrimaryAxis.Add(primaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![CategoryAxis support in MAUI Chart](Axis_Images/MAUI_category_axis.png)
+
+### Interval
+
+By default, the [CategoryAxis]() axis labels are display with fixed interval 1. It can customized by using the [Interval]() property of axis.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfPolarChart>
+    . . .
+    <chart:SfPolarChart.PrimaryAxis>
+        <chart:CategoryAxis Interval="2" />
+    </chart:SfPolarChart.PrimaryAxis>
+    <chart:SfPolarChart.SecondaryAxis>
+        <chart:NumericalAxis/>
+    </chart:SfPolarChart.SecondaryAxis>
+
+</chart:SfPolarChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfPolarChart chart = new SfPolarChart();
+. . .
+CategoryAxis primaryAxis = new CategoryAxis()
+{
+    Interval = 2, 
+};
+chart.PrimaryAxis.Add(primaryAxis);
+
+NumericalAxis secondaryAxis = new NumericalAxis();
+chart.SecondaryAxis.Add(secondaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## DateTime Axis
+
+[DateTimeAxis]() is used to plot `DateTime` values. The [DateTimeAxis]() is widely used to make financial charts in places like the Stock Market, where index plotting is done every day.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfPolarChart>
+    . . .
+    <chart:SfPolarChart.PrimaryAxis>
+        <chart:DateTimeAxis/>
+    </chart:SfPolarChart.PrimaryAxis>
+
+</chart:SfPolarChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfPolarChart chart = new SfPolarChart();
+. . .
+DateTimeAxis primaryAxis = new DateTimeAxis();
+chart.PrimaryAxis.Add(primaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![DateTimeAxis support in MAUI Chart](Axis_Images/MAUI_datetime_axis.png)
+
+### Interval
+
+In [DateTimeAxis](), intervals can be customized by using the [Interval]() and [IntervalType]() properties. For example, setting [Interval]() as 6 and [IntervalType]() as [Months]() will consider 6 months as interval.
```

Comment:
In [DateTimeAxis](), intervals can be customized using the [Interval]() and [IntervalType]() properties. For example, setting [Interval]() as 6 and [IntervalType]() as [Months]() will consider 6 months as interval.